tiktok.cs

```diff
@@ -5,394 +5,105 @@ namespace First_project
-                if (board[xGuess, yGuess - 1] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            return false;
-        }
-
-            static void Main(string[] args)
-        {
-
-            
             bool play = true;
-            string playString;
-            while (play == true)
+            while (play)
             {
 
                 int xGuess;
                 int yGuess;
-                int[,] newBoard = new int[,] { { 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0 } };
+                int[,] newBoard = new int[,] { {0, 0, 0}, {0, 0, 0}, { 0, 0, 0} };
                 int[,] board = newBoard;
-                int turnCounter = 1;
-                while (turnCounter <= 9)
-                {
-                    
-                    //XTurn
-                    XTurn(out xGuess, out yGuess);
-
-                    if (xGuess > 3)
-                    {
-                        Console.WriteLine("That's not a square! Try again.");
-                        XTurn(out xGuess, out yGuess);
-                    }
-                    if (xGuess < 1)
-                    {
-                        Console.WriteLine("That's not a square! Try again.");
-                        XTurn(out xGuess, out yGuess);
-                    }
-                    else
-                    {
-
-                        if (board[xGuess, yGuess] != 0)
-                        {
-                            Console.WriteLine("Already used this square! Pick another.");
-                            XTurn(out xGuess, out yGuess);
-                        }
-                        else
-                        {
-                            board[xGuess, yGuess] = 1;
-                            DrawBoard(board);
-                            if (CheckVictoryX(board, xGuess, yGuess) == true)
-                                {
-                                goto End;
-                            }
-
-
-                        }
-                    }
-                    turnCounter += 1;
-                    //YTurn
-                    YTurn(out xGuess, out yGuess);
-
-                    if (xGuess > 3)
-                    {
-                        Console.WriteLine("That's not a square! Try again.");
-                        YTurn(out xGuess, out yGuess);
-                    }
-                    if (xGuess < 1)
-                    {
-                        Console.WriteLine("That's not a square! Try again.");
-                        YTurn(out xGuess, out yGuess);
-                    }
-                    else
+                int turnCounter = 0;
+                bool invalidTurn = false;
+                bool draw = true;
+                DrawBoard(board);
+                while (turnCounter <= 8)
+                {
+                    int playerToken = turnCounter % 2 + 1;

```

Comment:
We can know which player it is based on turnCounter, that avoids the repeated methods for X and Y above.